ci/README.md

```diff
@@ -0,0 +1,119 @@
+# Testing Write and Read instance
+
+## Fake Replication Setup
+NOTE: This setup is enough to test basic route functions with Read & Write instances. Scroll down if you need fully functional / real time replication between read and write instances.
+####1. Setup Docker-Compose
+Update local environment in your `.docker-env`. You should have Read & Write URIs pointing to different instances.
+```
+SQLALCHEMY_DATABASE_URI=postgresql://postgres:LocalPassword@db:5432/notification_api
+SQLALCHEMY_DATABASE_URI_READ=postgresql://postgres:LocalPassword@db-read:5432/notification_api
+```
+Start containers
+```
+docker-compose -f ci/docker-compose-local-readwrite.yml up -d
+```
+
+####2. Create snapshot of the write database instance
+```
+pg_dump -U postgres -W -F t notification_api > /var/db-data/writedb.dump
+```
+
+####3. Copy snapshot of the write database instance into read instance's mounted volume on your system
+```
+cp ci/db-data/writedb.dump ci/db-data-read/writedb.dump
+```
+
+####4. Restore data from write instance on read instance
+```
+pg_restore -U postgres -W -d notification_api /var/db-data-read/writedb.dump
+```
+
+####5. Stop containers & remove data when done testing
```

Comment:
Some formatting issues with this. Please check that all your formatting is good with this doc.
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/16893311/287cba20-f726-4775-b044-597ff79c91d3)